conbench/entities/history.py

```diff
@@ -12,16 +12,36 @@
 class _Serializer(EntitySerializer):
     def _dump(self, history):
         standard_deviation = history.mean_sd if history.mean_sd else 0
+
+        # Note(JP): expose `times` or `data` or flatten them or expose both?
+        # Unclear. `times` is specified as "A list of benchmark durations. If
+        # data is a duration measure, this should be a duplicate of that
+        # object." `data` is specified with "A list of benchmark results (e.g.
+        # durations, throughput). This will be used as the main + only metric
+        # for regression and improvement. The values should be ordered in the
+        # order the iterations were executed (the first element is the first
+        # iteration, the second element is the second iteration, etc.). If an
+        # iteration did not complete but others did and you want to send
+        # partial data, mark each iteration that didn't complete as null."
+        # Expose both for now.
+        #
+        # In practice, I have only seen `data` being used so far and even when
+        # `data` was representing durations then this vector was not duplicated
+        # as `times`.
+
         return {
             "benchmark_id": history.id,
             "case_id": history.case_id,
             "context_id": history.context_id,
             "mean": float(history.mean),
+            "data": [float(d) if d is not None else None for d in history.data],
+            "times": [float(t) if t is not None else None for t in history.times],
```

Comment:
@austin3dickey I'd love to have your careful analysis on these two lines: do we think that they can break in view of existing data? As you can see, I rely on values being either floaty or None.

Reply:
Yes, I believe the column itself is nullable, so you might need to handle the case where `history.data` is None instead of a list, too. Maybe just treat it as an empty list in that case.

Also, if you'd like you can use `from ..entities._entity import to_float`.

Reply:
done